kong/core/globalpatches.lua

```diff
@@ -204,10 +218,11 @@ return function(options)
       -- this version of the randomseeding patch is required for 
       -- production, but doesn't work in tests, due to the ffi dependency
       local util = require "kong.tools.utils"
-      local seed
+      local seeds = {}
       local randomseed = math.randomseed
 
       _G.math.randomseed = function()
+        local seed = seeds[ngx.worker.pid()]
         if not seed then
           if not options.cli and ngx.get_phase() ~= "init_worker" then
             ngx.log(ngx.WARN, "math.randomseed() must be called in init_worker ",
```

Comment:
Could we also update this to call it with the error message as a 'debug.traceback' argument like above?